resources/pool.rb

```diff
@@ -21,19 +21,60 @@
 
 actions :add, :config, :delete, :start, :stop, :restart, :recycle
 
+# root
 attribute :pool_name, :kind_of => String, :name_attribute => true
-attribute :runtime_version, :kind_of => String
 attribute :no_managed_code, :kind_of => [TrueClass, FalseClass], :default => false
 attribute :pipeline_mode, :kind_of => Symbol, :equal_to => [:Integrated, :Classic]
-attribute :private_mem, :kind_of => Integer
-attribute :worker_idle_timeout, :kind_of => String
-attribute :recycle_after_time, :kind_of => String
-attribute :recycle_at_time, :kind_of => String
-attribute :max_proc, :kind_of => Integer
+attribute :runtime_version, :kind_of => String
+
+# add items
+attribute :start_mode, :kind_of => Symbol, :equal_to => [:AlwaysRunning, :OnDemand], :default => :OnDemand
+attribute :auto_start, :kind_of => [TrueClass, FalseClass], :default => true
+attribute :queue_length, :kind_of => Integer, :default => 1000
 attribute :thirty_two_bit, :kind_of => Symbol
+
+# processModel items
+attribute :max_proc, :kind_of => Integer
+attribute :load_user_profile, :kind_of => [TrueClass, FalseClass], :default => false
 attribute :pool_identity, :kind_of => Symbol, :equal_to => [:SpecificUser, :NetworkService, :LocalService, :LocalSystem, :ApplicationPoolIdentity ], :default => :ApplicationPoolIdentity
 attribute :pool_username, :kind_of => String
 attribute :pool_password, :kind_of => String
+attribute :set_profile_environment, :kind => [TrueClass, FalseClass], :default => true
+attribute :logon_type, :kind => Symbol, :equal_to => [:LogonBatch, :LogonService], :default => :LogonBatch
```

Comment:
@EasyAsABC123 I think we are getting errors on the 2 lines above.  Assuming :kind should be :kind_of?